flaml/default/greedy.py

```diff
@@ -12,7 +12,7 @@ def _augment(row):
 def construct_portfolio(regret_matrix, meta_features, regret_bound):
     """The portfolio construction algorithm.
 
-    (Reference)[https://arxiv.org/abs/2202.09927].
+    See: `Mining Robust Default Configurations for Resource-constrained AutoML <https://arxiv.org/abs/2202.09927>`_.
```

Comment:
This won't work.
![image](https://github.com/microsoft/FLAML/assets/3197038/60ccd661-7cef-4c32-bedd-d0a1cfe3383b)
```suggestion
    Reference: [Mining Robust Default Configurations for Resource-constrained AutoML](https://arxiv.org/abs/2202.09927).
```

Reply:
![image](https://github.com/microsoft/FLAML/assets/3197038/54d54caa-4a87-4a04-9003-6eda77bdb17f)